README.md

```diff
@@ -19,52 +19,65 @@ YourAppDelegate.h
 
 YourAppDelegate.m
 ```objective-c
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+    EDQueueStorageEngine *fmdbBasedStorage = [[EDQueueStorageEngine alloc] initWithName:@"database.sample.sqlite"];
+
+    self.persistentTaskQueue = [[EDQueue alloc] initWithPersistentStore:fmdbBasedStorage];
```

Comment:
Eh. Category is not a good idea :) 

So, let's keep the bound with FMDB. Introduced `defaultQueue`